### PR TITLE
[QP-1] login using valid credentials

### PR DESCRIPTION
**Pull request by dhanunjayaakula**
## **Description** 

This ticket is mainly for demo purpose. Please add detailed description of your work. So that who ever is Reviewing your work will get the proper context.

## Results
Please add your reports of the TC and if possible screen shots of the tests.

### Links
Link your XD's, Jira tickets..

### CODE REVIEW

**Review by Anirudhpu — APPROVED** [review]
Lgtm

**Review by Anirudhpu — DISMISSED** [review]
LGTM

File: Scripts/WEB/gggg/Script1669238669558.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import static com.kms.katalon.core.checkpoint.CheckpointFactory.findCheckpoint
+import static com.kms.katalon.core.testcase.TestCaseFactory.findTestCase
```
Comment (Anirudhpu) [inline Scripts/WEB/gggg/Script1669238669558.groovy:2]: This should be changed

**Review by Anirudhpu — COMMENTED** [review]
Left some comments

File: Scripts/WEB/gggg/Script1669238669558.groovy (hunk 1 above)
Comment (dhanunjayaakula) [reply Scripts/WEB/gggg/Script1669238669558.groovy:2]: It is for demo test you can approve

**Review by Anirudhpu — APPROVED** [review]
LGTM

**Pull request merged**